extension/goplugin/schemas.go

```diff
@@ -523,6 +523,11 @@ func (schema *Schema) Properties() []goext.Property {
 	return properties
 }
 
+// Parents returns schema_ids which given schema extends
+func (schema *Schema) Parents() []string {
+	return schema.raw.Extends
```

Comment:
In gohan, it it named 'extends' but in the interface it is parent. I think it's an inconsistency that should be fixed.

Reply:
I've changed that to keep consistency, but Extends doesn't fit to me at all.